### ISSUE TITLE
Did winter time change event times?

### ISSUE THREAD

**Issue by tadoh** [ronan-mch/det_sker#383]
My event series Folkets Haus changed time after we moved to winter time. It now appears from 17 to 22.45 instead of being from 18 to 23.45.

What to do?

**Comment by tadoh** [comment]
https://www.dukop.dk/users/16?locale=da

**Comment by benjaoming** [comment]
Her er et andet eksempel: Man opretter et gentagende event, hvor serien overlapper med skift til sommertid.

Løsning 1: Ny besked til brugeren: "NB! Din serie overlapper med skift mellem sommer- og vintertid, og du bør kigge den efter manuelt".

Løsning 2: Når selve skiftet i sommer/vintertid sker, burde serien automatisk tilpasse tidspunkter for events i den del af serien, der ikke har samme DST, som udgangspunktet.

![screenshot from 2016-12-25 14-14-42](https://cloud.githubusercontent.com/assets/374612/21471279/c704863a-caac-11e6-9a02-19e4f11e55be.png)

**Comment by benjaoming** [comment]
Løsning 1 fungerer, fordi man kan klikke sig ind på de enkelte events i serien og lave individuelle tilrettelser.

**Comment by Siggik** [comment]
Men løsning 2 ville være at foretrække, right?

**Comment by benjaoming** [comment]
@Siggik ja, det ville være at foretrække, fordi man med det samme man opretter eventet får det ønskede resultat :)
